docs/build.sh

```diff
@@ -15,18 +15,33 @@ fi
 
 # We need to set ENVOY_DOCS_VERSION_STRING and ENVOY_DOCS_RELEASE_LEVEL for Sphinx.
 # We also validate that the tag and version match at this point if needed.
-VERSION_NUMBER=$(cat VERSION)
+VERSION=$(cat VERSION)
```

Comment:
Sorry can you explain this part more? I'm confused. Shouldn't the docs just be snapped to the versions that are going to be published? I don't think this should be the last version?